api/scripts/requireAcceptToS.js

```diff
@@ -0,0 +1,56 @@
+const {
+  RequiredActionAlias,
+} = require("keycloak-admin/lib/defs/requiredActionProviderRepresentation");
+require("dotenv").config({ path: "api/.env" });
+const initKcAdminClient = require("../utils/initKcAdminClient");
+
+if (!process.env.KEYCLOAK_AUTH_SERVER) {
+  throw ".env not loaded. Call from the top dir in the repo";
+}
+
+const TERMS = RequiredActionAlias.terms_and_conditions;
```

Comment:
We should also add this required action in the `inviteEventMembers` and `inviteOrgMembers` resolvers

Reply:
I believe that would probably be taken care of by the setting in keycloak that requires the ToS to be accepted by new users (that I've enabled). But maybe we need to test that.

Reply:
Tested it now, it becomes part of the "Complete signup" process as the last step. It's not listed as one of the steps on the first screen shown after clicking that link though

Reply:
Ok good. That's how the ToS accepting works in keycloak by default. You accept it on the screen after registering/loggin in. It's a bit unusual but works. I think I read you could technically maybe hack in a checkbox on the registering screen but I don't know if that would be worth the effort.

Can I mark this as resolved?